applications/MultilevelMonteCarloApplication/tests/Level0/ProjectParameters.json

```diff
@@ -24,25 +24,25 @@
             "solver_type": "convection_diffusion_base_solver",
             "model_import_settings": {
                 "input_type": "mdpa",
-                "input_filename": "../tests/Level0/Level0"
+                "input_filename": "/home/kratos105b/Kratos/applications/MultilevelMonteCarloApplication/tests/Level0/Level0"
```

Comment:
This is not a good idea

Reply:
+1 for @loumalouomega 's comment. this should be relative

Reply:
Ramon suggested me that it is always better to use the absolute path when using compss, I can restore the relative path and add a warning. Do you agree?

Reply:
no, as it will not work in a computer that is not yours.

for the version in github you need to default to pure python and use that in tests